jumanji/environments/routing/robot_warehouse/env.py

```diff
@@ -491,7 +491,8 @@ def reward_and_update_request_queue_if_shelf_in_goal(
             return key, reward, request_queue, shelves
 
         # check if shelf is at goal position and in request queue
-        cond = (shelf_id != 0) & jnp.isin(shelf_id, request_queue + 1)
+        cond = jnp.logical_not(jnp.equal(shelf_id, 0))
+        cond = jnp.logical_and(cond, jnp.isin(shelf_id, request_queue + 1))
```

Comment:
Why is this code being changed? The original looks better.

Reply:
Reverted to Sasha's version